### PR TITLE
Initial check imports script

### PR DESCRIPTION
**Pull request by paulb777**
Verify functional and style of imports and includes.

### CODE REVIEW

File: scripts/check_imports.swift [hunk 1]
```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/swift
```
Comment (wilhuff) [inline scripts/check_imports.swift:1]: Writing this kind of check in Swift means these checks can't run on Windows. While most of Firebase doesn't support Windows from this repo, Firestore does. Please reconsider this.

File: scripts/check_imports.swift [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+/*
+ * Copyright 2020 LLC Google
```
Comment (wilhuff) [inline scripts/check_imports.swift:3]: Google LLC

File: scripts/check_imports.swift (hunk 1 above)
Comment (paulb777) [reply scripts/check_imports.swift:1]: Swift looks to be coming soon to Windows - next week perhaps?  https://swift.org/blog/5-3-release-process/.

File: scripts/check_imports.swift (hunk 2 above)
Comment (paulb777) [reply scripts/check_imports.swift:3]: Thanks

**Comment by paulb777** [comment]
Maybe?  

I'm prioritizing more integration testing first in expanding the script's coverage across the repo first.

**Pull request merged**